src/scss/use-cases/_general.scss

```diff
@@ -154,6 +154,10 @@
 		));
 		@include oTypographyMargin($top: 0, $bottom: 0);
 		color: oColorsGetColorFor('body', 'text');
+
+		> p {
+			@include oTypographyMargin($top: 0, $bottom: 0);
```

Comment:
I'm not sure this should be enforced by o-typography -- what if a list item contained multiple paragraphs?

Reply:
Hmm. That is an excellent point.

I guess doing some `first-of-type` and `last-of-type` would resolve it. But then that's even more niche...

Reply:
What's the usecase -- is the single `p` tag redundant?

Reply:
We could use `:only-child` but if the `p` element isn't needed it would be nice to save a few bytes of css.

Reply:
To be honest, I'm not sure how valid the use case is.

* Single p tags inside an li : this is reasonable but redundant, so we could strip this out.
* Multiple p tags inside an li: Sounds like something that could happen, but not sure how often.

For multiple p tags, I'm not sure :only-child would cut it. The margins on the multiple p tags would separate that out from the rest of the list.

![Screenshot 2019-06-07 at 13 38 18](https://user-images.githubusercontent.com/1978880/59104593-dca55d00-8929-11e9-9b9c-aa98c55c0860.png)

Reply:
>For multiple p tags, I'm not sure :only-child would cut it. The margins on the multiple p tags would separate that out from the rest of the list.
Good point. If we removed the top margin from the first and bottom from the last, that would also look weird.

>Single p tags inside an li : this is reasonable but redundant, so we could strip this out.
So maybe option a?

Reply:
Yeahhhh okay. I'll close this, thanks for indulging me 🙇

Reply:
Thank you for the suggestion and PR! :)